### PR TITLE
Results overview layout

### PR DESCRIPTION
**Pull request by RCode-Blue**
## Overview of the Pull Request:
Edited layout to match other sections in the _Results_ page

## link to Trello card or Github issue
https://trello.com/c/vl2rpehH/322-results-page-has-multiple-layout-methods-which-looks-odd-on-wide-browser-windows

## How Has This Been Tested?
The right-indentation for the _Overview_ section should match that of the other sections:
(Guide borders have been removed from commit)

image

image

## Pull Request Checklist:
- [X] Have you included a link Trello card / Github issue and written sufficient description to help others review your work?
- [X] Is your pull request up-to-date with `main` branch?
- [X] Have you checked That code is well formatted? Did `npx prettier --check .` return no warnings/errors?
- [X] Have you written instructions on how to test that your changes work as expected?
- [X] Have you tested your work thoroughly on your local machine to ensure you are not breaking anything?

### CODE REVIEW

**Comment by hqtan** [comment]
Hi @RCode-Blue ,

It looks like the changes in this PR may have caused the Intro section to be 'misaligned' with the 'Overview' section:
image

This is currently how it looks like in [prod](https://seechange.codeforaustralia.org/results): 
image

**Comment by RCode-Blue** [comment]
Sorry @hqtan  my bad. 

Is it better now?

File: components/ResultsPageComponents/CommuteDays/CommuteDays.jsx [hunk 1]
```diff
@@ -14,6 +14,7 @@ export default function CommuteDays({ data }) {
       px={["5px", "50px"]}
       py={["25px", "50px"]}
       justify="center"
+      border="1px dashed red"
```
Comment (hqtan) [inline components/ResultsPageComponents/CommuteDays/CommuteDays.jsx:17]: please remove the dashed border
```suggestion
```

**Review by hqtan — CHANGES_REQUESTED** [review]
Hi @RCode-Blue ,

Thx for your PR 🙇🏽 

The changes in this PR are marginal, but I think they do offer a *slight* improvement, so happy to approve it, after the suggested changes are made.

**Review by OlaShabalina — APPROVED** [review]
Well done @RCode-Blue !

**Review by hqtan — APPROVED** [review]
lgtm @RCode-Blue 👍🏽 

let's ship it :shipit:

**Pull request merged**